Soccar [UI]/src/nl/socnet/message/handler/JoinSessionMessageHandler.java

```diff
@@ -18,16 +19,32 @@
 
 /**
  * Handles JoinSession messages, after the currentPlayer has joined a session.
- * 
+ *
  * @author PTS34A
  */
 public final class JoinSessionMessageHandler extends MessageHandler<JoinSessionMessage> {
 
     @Override
     protected void handle(Connection connection, JoinSessionMessage message) throws Exception {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "handle" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)